azurerm/internal/services/web/parse/static_site.go

```diff
@@ -0,0 +1,33 @@
+package parse
+
+import (
+	"fmt"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+)
+
+type StaticSiteResourceID struct {
+	ResourceGroup string
+	Name          string
+}
+
+func StaticSiteID(input string) (*StaticSiteResourceID, error) {
```

Comment:
Could we have a test for this function as for other ID type elsewhere in the provider?